CedarJava/src/main/java/com/cedarpolicy/model/AuthorizationSuccessResponse.java

```diff
@@ -95,6 +95,29 @@ public AuthorizationSuccessResponse(
         this.diagnostics = diagnostics;
     }
 
+    /** Error (or warning) which occurred in a particular policy during authorization */
```

Comment:
Nit: Currently there are no warnings returned at authorization time. Warnings would instead be captured in the `AuthorizationResponse` struct.